Azure/sonic-buildimage
# DellEMC: Z9332f media settings

#### Why I did it
Common method is needed to configure pre-emphasis settings in z9332f platform. 
#### How I did it
Changed DellEMC Z9932f media settings from Vendor Name + PN method to common method.

#### How to verify it
Check APP_DB PORT_TABLE: EthernetX and verify the  pre-emphasis settings in NPU.
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog
UT:
[media_settings_UT.txt](https://github.com/Azure/sonic-buildimage/files/6400398/media_settings_UT.txt)


#### A picture of a cute animal (not mandatory but encouraged)
